test/bigtest/tests/units/settings/MappingProfileDetails/MappingProfileDetails-test.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+import { noop } from 'lodash';
+import sinon from 'sinon';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { Paneset } from '@folio/stripes/components';
+import { mountWithContext } from '@folio/stripes-data-transfer-components/interactors';
+import commonTranslations from '@folio/stripes-data-transfer-components/translations/stripes-data-transfer-components/en';
+import stripesComponentsTranslations from '@folio/stripes-components/translations/stripes-components/en';
+
+import translations from '../../../../../../translations/ui-data-export/en';
+import { MappingProfileDetails } from '../../../../../../src/settings/MappingProfiles/MappingProfileDetails';
+import { MappingProfileDetailsInteractor } from './interactors/MappingProfileDetailsInteractor';
+
+describe('MappingProfileDetails', () => {
```

Comment:
It seems, I missed something and wasn't able to find check that the metadata is in place. Please, could you clarify if it is in place.

Reply:
Yes, I forgot about it, thanks for noticing, already updated